components/brave_rewards/resources/rewards_panel/components/publisher_card.tsx

```diff
@@ -97,6 +97,28 @@ export function PublisherCard () {
     return publisherInfo.name
   }
 
+  function renderPublisherIcon () {
+    if (!publisherInfo || !publisherInfo.icon) {
+      return null
+    }
+
+    if (publisherInfo.platform) {
+      return (
+        <span className='rounded'>
+          <style.icon>
+            <img src={publisherInfo.icon} />
+          </style.icon>
+        </span>
+      )
+    }
+
+    return (
+      <style.icon>
```

Comment:
Just FYI, but you could put the class name in here if you wanted to, like:

```tsx
<style.icon className={publisherInfo.platform ? 'rounded' : ''}>
   ...
</style.icon>
```

Reply:
Thank you, that's much more concise!